### PR TITLE
Census Individual: NISRA: Content changes - Work/School

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
Content changes for NISRA

### How to review 
Ensure:
- Changes are as per instructions on Trello.

[Quick launch for NIR schema](https://v3-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HI&region_code=GB-NIR&url=https://raw.githubusercontent.com/ONSdigital/eq-survey-runner/eq-3143-nisra-content-changes-job-m/data/en/census_individual_gb_nir.json)

### CODE REVIEW

File: data/en/census_individual_gb_nir.json [hunk 1]
```diff
@@ -0,0 +1,8443 @@
+{
```
Comment (MebinAbraham) [inline data/en/census_individual_gb_nir.json]: This is just for quick launch purposes and will be removed before squash.

**Comment by MebinAbraham** [comment]
[NISRA content changes - Job (M)](https://trello.com/c/F88H8AD2/3143-nisra-content-changes-job-m)

**Review by bambrp — COMMENTED** [review]
https://github.com/ONSdigital/eq-survey-runner/blob/befc62622d8711147d3e2db121820ff43062f701/data-source/jsonnet/northern-ireland/blocks/individual/employment/business_name.jsonnet#L45

pastNonProxyOption appears unused (for good reason), should it be deleted?

**Comment by MebinAbraham** [comment]
> https://github.com/ONSdigital/eq-survey-runner/blob/befc62622d8711147d3e2db121820ff43062f701/data-source/jsonnet/northern-ireland/blocks/individual/employment/business_name.jsonnet#L45
> 
> pastNonProxyOption appears unused (for good reason), should it be deleted?

Removed.

**Review by donnap573 — APPROVED** [review]

**Comment by bambrp** [comment]
https://github.com/ONSdigital/eq-survey-runner/blob/4173d87e7b4d7dcd2960713471d9c48c9e1d085b/data-source/jsonnet/england-wales/blocks/individual/employment/business_name.jsonnet#L40

work -> works

**Comment by MebinAbraham** [comment]
> https://github.com/ONSdigital/eq-survey-runner/blob/4173d87e7b4d7dcd2960713471d9c48c9e1d085b/data-source/jsonnet/england-wales/blocks/individual/employment/business_name.jsonnet#L40
> 
> work -> works

This PR is just for NISRA, don't want to change ENG/WLS content unless specified by the card.

**Review by bambrp — APPROVED** [review]
Everything looks implemented except one eng/wal change that's in another PR.

**Comment by MebinAbraham** [comment]
@donnap573 @bambrp Can you just recheck to ensure it still works as expected. Had to change implementation from blocks to groups.

**Review by bambrp — APPROVED** [review]
LGTM (Let's Get This Merged)

**Review by donnap573 — APPROVED** [review]

**Pull request merged**